src/dropdown.ts

```diff
@@ -183,6 +183,7 @@ export default class GlideCoreDropdown
     this.#isOpen = isOpen;
 
     if (isOpen && hasChanged && !this.disabled) {
+      this.isNoAvailableOptions = this.#optionElements.length === 0;
```

Comment:
A getter would be nice. But a getter without a setter won't trigger a rerender. Same for `isNoMatchingOptions` below.